config/routes.rb

```diff
@@ -278,6 +262,9 @@ def not_implemented
     end
   end
 
-  match '*path', via: :all, to: not_implemented
-
+  if Feature.active?(:redirects)
+    match '*path', via: :all, to: 'catchall#not_found'
```

Comment:
It is, but for the sake of legacy code readability. Additionally, there is not 404 ```not_found``` template on frontend. Not unless you will be rendering 404's through CMS